### PR TITLE
feat: Dynamic Rows

### PR DESCRIPTION
**Pull request by timarney**
# Summary | Résumé

Re-enables dynamic row feature for CDS admin users

### CODE REVIEW

**Review by thiessenp-cds — DISMISSED** [review]
LGTM :)

**Comment by timarney** [comment]
Holding off on merging this until after the Conditional Logic updates go live

File: lib/utils/form-builder/getPath.ts [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
   // const idStr = id.toString();
   // const sliceAt = idStr.length == 3 ? 1 : 2;
   // const idArr = idStr.split("");
   // const first = idArr.slice(0, sliceAt).join("");
+  // return Number(first);
   return Number(id);
 };
 
+export const parseParentId = (id: number, elements: Element[]) => {
+  elements.forEach((el) => {
```
Comment (thiessenp-cds) [inline lib/utils/form-builder/getPath.ts:17]: May want to use a var vs. param to store/update the id

**Review by thiessenp-cds — APPROVED** [review]
Gave it a quick tests. Noticed a few small things, like some contrast issues in the UI but anything else can be for a follow up PR. LGTM :)

**Review by dsamojlenko — APPROVED** [review]

**Pull request merged**